### DISCUSSION TITLE
win11管理员权限问题

### DISCUSSION THREAD

**Discussion by yumeiyan77** [Planshit/Tai#57] [Q&A]
作者你好，这个软件用到现在我非常满意，非常感谢作者。我想向你问个问题，我也不清楚是不是软件的问题，有些软件使用时长无法统计需要使用管理员权限，但是设置管理员权限后，开机自启却失效了，无法开机自启，手动点击能够开启软件，该如何解决呢，系统是win11。

**Comment by noberumotto** [comment]
谢谢反馈。当前版本暂无解决方案，下一个版本会解决这个问题

**Comment by Xrysnow** [comment]
我也遇到这个问题了，查资料说用计划任务可以自启，希望软件里能增加这个功能

**Reply by noberumotto** [reply]
已解决此问题，请更新版本

**Comment by noberumotto** [comment]
已解决此问题，请更新版本
